src/main/java/sirius/biz/jupiter/JupiterDataProvider.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.jupiter;
+
+import java.io.OutputStream;
+
+/**
+ * Provides data to be stored in a file inthe local Jupiter repository.
+ * <p>
+ * The {@link JupiterSync} will invoke all data providers each night to update their file contents in the local
+ * repository. This will then be synchronized with the repository in the Jupiter instance itself so that
+ * InfoGraphDB and other consumers can load this data.
+ */
+public interface JupiterDataProvider {
```

Comment:
AutoRegister?

Reply:
Interfaces are "inherently autoregister" - as long as we don't plan to use a common abstract base class, this isn't strictly required - on the other hand, it doesn't hurt

Reply:
...and it also signals that it is intended to be registered...